kernel/fs/fs.go

```diff
@@ -21,8 +21,13 @@ import (
 	"tractor.dev/wanix/internal/mountablefs"
 )
 
+var DebugLog string
+var doLogging bool = DebugLog == "true"
```

Comment:
Since this is only used in log, it could be inlined there to avoid extra "do" names (which I try to avoid)

Reply:
I chose this way since I'd rather not do string comparisons every time log is invoked. It doesn't seem like I can turn this into a constant expression either even though it should be resolvable at compile time.

Reply:
Ok that's fair. This is quick and dirty debug logging anyway, we can come back to this later to clean up.